### DISCUSSION TITLE
通知一覧に「コメントだけ表示」などの絞り込み機能がほしい

### DISCUSSION THREAD

**Discussion by ko1nksm** [increments/qiita-discussions#320] [要望・アイデア・不具合]
「いいね」や「ストック」の通知が多いと「コメントされた」という通知が流れてしまい、後から返信しようと思った時に見つけづらく忘れてしまいます。

「コメントだけ表示」もしくは「いいね・ストック以外を表示」と言った絞り込み機能、もしくは一覧のページを分けるなど、コメントされたという通知（や重要度の高い通知）を見つけやすくして欲しいです。

**Comment by tanaka-asami415** [comment]
@ko1nksm 
ご意見ありがとうございます🙏
直近の開発優先度などから考えると先になってしまうかもしれませんが、
コメントの他にも編集リクエストを見落とさないようになど、
社内でも何度か議論に上がっており、通知周りは改善したいと思っております！

**Reply by P-SiZK** [reply]
@ko1nksm 
大変お待たせしました :bow: 
通知をカテゴリーごとに確認できる機能を追加しました。
https://qiita.com/release-notes/2024-09-11

コメント・メンションのタブで絞り込まれた通知を以下 URL から確認できます！
https://qiita.com/notifications?scope=comment_and_mention

**Reply by tanaka-asami415** [reply]
大変申し訳ありません。
ご報告したばかりなのですが、一部のユーザー様において表示が重い状況が発生しており、もしかすると表示されない事があるかもしれません。

現在修正中なので、完了したら改めてご報告いたします🙇
https://x.com/Qiita/status/1836678416395047139

**Reply by ko1nksm** [reply]
対応ありがとうございます。機能が追加されているのに気づき、
これは便利だと思っていたら、私が要望していたんですね。忘れていました😁
重い状況は了解です。

**Reply by tanaka-asami415** [reply]
早速ご利用いただきありがとうございます！
修正を行い、問題なくご利用いただける状態となりました🙇
https://x.com/Qiita/status/1838396877546676301

リクエストいただきありがとうございました✨
